packages/slider/package.json

```diff
@@ -24,7 +24,8 @@
         "./src/*": "./src/*",
         "./package.json": "./package.json",
         "./sp-slider": "./sp-slider.js",
-        "./sp-slider.js": "./sp-slider.js"
+        "./sp-slider.js": "./sp-slider.js",
+        "./sp-slider-handle.js": "./sp-slider-handle.js"
```

Comment:
We should include the no extension on the left version as well.

Reply:
Good catch!